.github/gitlint/contrib_description_conventional_commits.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from gitlint.rules import CommitRule, RuleViolation
+
+
+class SignedOffBy(CommitRule):
```

Comment:
Why is this called signed off by? It has nothing to do with signing

Reply:
Because I love to copy paste brainlessly.

Reply:
SignOffBy means `git commit -s ...`

Also, yes it has nothing to-do with cryptographic signing.

Git doc ref:
https://git-scm.com/docs/git-commit#Documentation/git-commit.txt--s

Commit conventions:
https://git.wiki.kernel.org/index.php/CommitMessageConventions